lib/inventory_refresh/inventory_collection.rb

```diff
@@ -232,16 +227,16 @@ def unique_index_keys
 
     # @return [Array<ActiveRecord::ConnectionAdapters::IndexDefinition>] array of all unique indexes known to model
     def unique_indexes
-      @unique_indexes_cache if @unique_indexes_cache
+      @unique_indexes if @unique_indexes
```

Comment:
@agrare Are you fixing this in a follow up?

Reply:
Yes I split this out into its own PR since it wasn't related to the rubocop changes specifically